lib/src/integrationTest/kotlin/com/lemonappdev/konsist/core/declaration/kofiledeclaration/KoFileDeclarationForKoAnnotationProviderTest.kt

```diff
@@ -1,4 +1,4 @@
-package com.lemonappdev.konsist.core.container.kofile
+package com.lemonappdev.konsist.core.declaration.kofiledeclaration
 
```

Comment:
I think we should add new ticket to change all packages like this `com.lemonappdev.konsist.core.declaration.kofiledeclaration`to `com.lemonappdev.konsist.core.declaration.kofile`
<img width="334" alt="Pasted Graphic" src="https://github.com/LemonAppDev/konsist/assets/111683562/c7048963-d2a6-455c-a6db-1d1db1446aac">

Reply:
I create ticket for this